posthog/session_recordings/realtime_snapshots.py

```diff
@@ -91,6 +97,7 @@ def get_realtime_snapshots(team_id: str, session_id: str, attempt_count=0) -> Op
                     snapshots.append(line.decode("utf8"))
 
             REALTIME_SUBSCRIPTIONS_LOADED_COUNTER.labels(attempt_count=attempt_count).inc()
+            REALTIME_SUBSCRIPTIONS_DATA_LENGTH.labels(attempt_count=attempt_count).observe(len(snapshots))
```

Comment:
we've always been pushed away from adding team_id in prom series to avoid the cardinality explosion...

here i only really care about incidence of zeroes changing (and for attempt_count I could get away with `attempt_count == 0` but I know it has a limited number of values so 🤷